website/src/_posts/2020-08-companion-2.0.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Companion 2.0 is here"
+date: 2020-08-20
+author: ife
+published: false
+---
+
+Companion 2.0 is here! 🎉 After maintaining and improving the 1.x series for over a year, we're releasing a Major version bump on the Companion package. So what are the changes you'd expect to see on Companion 2.0?
+
+## Node >= v10
+
+As you must have heard, Node v8.x has reached EOL, and in turn Companion 2.0 drops support for Node v6.x, Node v8.x and now requires that you run at least Node v10.x.
```

Comment:
imho we can shorten this a bit and explain the acronym. I think the "official" spelling for node versions is "Node.js X.Y.Z", without a `v`, but people would understand either way :)
```suggestion
Node.js 8.x has reached End-of-life, and in turn Companion 2.0 drops support for Node.js 6.x, Node.js 8.x and now requires that you run at least Node.js 10.x.
```